presto-native-execution/src/test/java/com/facebook/presto/nativeworker/AbstractTestNativeGeneralQueries.java

```diff
@@ -1635,6 +1635,33 @@ public void testColumnFilter()
         assertQuery(session, format("SELECT c_boolean, c_bigint, c_double, c_varchar, c_varbinary FROM %s WHERE c_varbinary = to_ieee754_64(1)", tmpTableName));
     }
 
+    @Test
+    public void testUuid()
+    {
+        assertQuerySucceeds("SELECT uuid()");
```

Comment:
I assume these queries run entirely on coordinator. If that
is the case, then we are not testing the worker. Did these queries fail before?